deliverables/ruby_monk_problems/sum_of_cubes.rb

```diff
@@ -1,3 +1,9 @@
 def sum_of_cubes(a, b)
-  # Write your code here
+    sum = b**3
+  b -= 1
+  while b >= a
```

Comment:
Looks like there might be an opportunity to reduce the number of lines here.  Check this out:  

``` ruby
def sum_of_cubes(a, b)
  sum = 0
  while b > a
    sum += b**3
    b = b - 1
  end

  sum
end
```  

Well I guess that only removed one line, but that's the basic idea.  You've got the same operation happening once before the loop and then again inside the loop.  So you're repeating those instructions.  Could tighten it up just by tweaking the parameters of the loop a little bit.